### DISCUSSION TITLE
PCL2启动器老是消失，但相关文件还在，求大佬解惑pwq

### DISCUSSION THREAD

**Discussion by ChestnutPlumage** [Meloong-Git/PCL#1049] [提问] [answered]
![image](https://user-images.githubusercontent.com/122587720/215237214-348f28d1-35e5-433e-a498-5be28ba2e558.png)
然后每次这样后我就又得去下载2.3.0，然后密钥更新，也不是很频繁发生，但是过一段时间就会这样，哭。

**Comment by WForst-Breeze** [comment, marked as answer]
请将 PCL2 添加至相关杀软白名单中，此处以 Windows Defender 为例：[将排除项添加到 Windows 安全中心](https://support.microsoft.com/zh-cn/windows/%E5%B0%86%E6%8E%92%E9%99%A4%E9%A1%B9%E6%B7%BB%E5%8A%A0%E5%88%B0-windows-%E5%AE%89%E5%85%A8%E4%B8%AD%E5%BF%83-811816c0-4dfd-af4a-47e4-c301afe13b26)

**Reply by ChestnutPlumage** [reply]
谢谢你！，我刚刚照做了
![image](https://user-images.githubusercontent.com/122587720/215237737-b09bedb1-95b1-41a7-8b58-545cc0a686b6.png)
![image](https://user-images.githubusercontent.com/122587720/215237747-26e75ba8-770b-4fd8-8310-50aab1bcb263.png)
不过如果我更新pcl2，并且更改文件名后，请问需要重新去设定吗

**Reply by IF-desu** [reply]
为什么要用杀软呢？我电脑不用杀软6年了未曾有过病毒

**Reply by WTP016-CN** [reply]
> 为什么要用杀软呢？我电脑不用杀软6年了未曾有过病毒

用不用是个人选择（
就像我从来不用WD用火绒一样

**Reply by ChestnutPlumage** [reply]
> 为什么要用杀软呢？我电脑不用杀软6年了未曾有过病毒

我没有用过的，电脑没有下载任何杀软，唯一有的可能就是windows安全中心？

**Reply by ChestnutPlumage** [reply]
> 应该要的.jpg

好的，谢谢，后面更新时我会留意的
